apps/system/js/card.js

```diff
@@ -142,11 +143,16 @@
 
     var origin = app.origin;
     var frameForScreenshot = app.getFrameForScreenshot();
+    var displayUrl = '';
 
-    if (frameForScreenshot &&
+    if (app.isBrowser()) {
+      displayUrl = app.config.url || origin;
+    } else if(frameForScreenshot &&
         CardsHelper.getOffOrigin(frameForScreenshot.src, origin)) {
-      this.subTitle = CardsHelper.getOffOrigin(
-                        frameForScreenshot.src, origin);
+      displayUrl = CardsHelper.getOffOrigin(frameForScreenshot.src, origin);
+    }
+    if (displayUrl) {
+      this.subTitle = CardsHelper.getDisplayURLString(displayUrl);
```

Comment:
Could we avoid "enhancing" CardsHelper here? :/
I think our compromise is to deprecate CardsHelper and move it into Card.
i.e., I think it's better to do |this.getDisplayURLString()| here. What do you think?